iam/api-client/access.py

```diff
@@ -18,39 +18,12 @@
 https://cloud.google.com/iam/docs/granting-changing-revoking-access.
 """
 
-import argparse
 import os
 
 from google.oauth2 import service_account  # type: ignore
 import googleapiclient.discovery  # type: ignore
 
 
-# [START iam_get_policy]
-def get_policy(project_id: str, version: int = 1) -> dict:
-    """Gets IAM policy for a project."""
-
-    credentials = service_account.Credentials.from_service_account_file(
-        filename=os.environ["GOOGLE_APPLICATION_CREDENTIALS"],
-        scopes=["https://www.googleapis.com/auth/cloud-platform"],
-    )
-    service = googleapiclient.discovery.build(
-        "cloudresourcemanager", "v1", credentials=credentials
-    )
-    policy = (
-        service.projects()
-        .getIamPolicy(
-            resource=project_id,
-            body={"options": {"requestedPolicyVersion": version}},
-        )
-        .execute()
-    )
-    print(policy)
-    return policy
-
-
-# [END iam_get_policy]
-
-
 # [START iam_modify_policy_add_member]
```

Comment:
Are we planning to rewrite these samples too? If yes, then it's OK to remove the tests; otherwise I would suggest to keep tests for them and only remove tests that match the region_tags which are being removed.

Reply:
`iam_modify_policy_add_member` and `iam_modify_policy_remove_member` are also rewritten, I'll remove those in next commit.

I also migrated the two other samples that were left in `access.py`, had to rewrite one to support GAPIC, the other one didn't need any changes.